server/app/services/geo/AddressSuggestionGroup.java

```diff
@@ -20,6 +20,15 @@ public static Builder builder() {
 
   public abstract Address getOriginalAddress();
 
+  /** Returns an empty instance of an AddressSuggestionGroup */
+  public static AddressSuggestionGroup empty(Address originalAddress) {
+    return AddressSuggestionGroup.builder()
+        .setWellKnownId(0)
```

Comment:
I don't like setting this to 0, but cleaning this up is outside the scope of this PR.